docker-compose.yml

```diff
@@ -160,8 +160,29 @@ services:
         - action: sync
           path: ./src/interfaces/coral_web
           target: /app
-          ignore:
-            - node_modules/
+
```

Comment:
Do we need to watch the node_modules folder?

Reply:
good callout, will add again. Removed for testing an issue I had and forgot to add back